src/modules/simulation/gz_bridge/GZMixingInterfaceESC.hpp

```diff
@@ -35,6 +35,7 @@
 
 #include <lib/mixer_module/mixer_module.hpp>
 
+#include <gz/msgs.hh>
```

Comment:
Is this needed or here by accident?

Reply:
it's actually required. idk why previously it compiled fine without that but it's needed by the gz msg actuator . And this is cherry-picked from upstream https://github.com/PX4/PX4-Autopilot/commit/2d7838329602f163b24d4eefbeb91ac947568186